### PR TITLE
add space and model hub for semantic_similarity_with_bert example

### PR DESCRIPTION
**Pull request by vumichien**
As a part of hugs keras sprint, this example was updated with model hub and space demo

NLP

- semantic_similarity_with_bert

### CODE REVIEW

File: examples/nlp/md/semantic_similarity_with_bert.md [hunk 1]
```diff
@@ -322,7 +322,7 @@ with strategy.scope():
     # Freeze the BERT model to reuse the pretrained features without modifying them.
     bert_model.trainable = False
 
-    bert_output = bert_model(
+    bert_output = bert_model.bert(
```
Comment (fchollet) [inline examples/nlp/md/semantic_similarity_with_bert.md:325]: Why this change?

Comment (vumichien) [reply examples/nlp/md/semantic_similarity_with_bert.md:325]: Sorry, I forgot to clarify this change. When I tried to save and load the trained model I got this error `ValueError: The two structures don't have the same nested structure.`
I changed this line following the solution for this issue in the HugingFace repo https://github.com/huggingface/transformers/issues/3627

**Review by fchollet — APPROVED** [review]
LGTM, thanks!
